backend/rs/ic_vetkeys/src/utils/mod.rs

```diff
@@ -589,6 +586,36 @@ impl IBECiphertext {
     }
 }
 
+/// Verify an augmented BLS signature
+///
+/// Returns true if and only if the provided signature is valid with respect to
+/// the provided public key and input
+pub fn verify_bls_signature(dpk: &DerivedPublicKey, input: &[u8], signature: &[u8]) -> bool {
+    let signature: G1Affine = match signature.try_into() {
```

Comment:
nit: slight preference for being explicit about what we are trying to convert into:
```
let signature = match <[u8; 48]>::try_from(signature) {
```

Reply:
Done